protocol/x/clob/keeper/process_operations.go

```diff
@@ -566,6 +581,25 @@ func (k Keeper) PersistMatchLiquidationToState(
 				),
 			)
 		}
+
+		// Stat fill amount in quote quantums as ratio of notional quote quantums
+		fillAmountToTotalSizeRat := new(big.Rat).Quo(
+			new(big.Rat).SetUint64(matchWithOrders.FillAmount.ToUint64()),
+			new(big.Rat).SetUint64(matchLiquidation.TotalSize),
+		)
+		filledQuoteQuantums := lib.BigRatRound(
+			new(big.Rat).Mul(
+				fillAmountToTotalSizeRat,
+				new(big.Rat).SetInt(notionalQuoteQuantums),
```

Comment:
What happens if `notionalQuoteQuantums` is negative?

Note that the return value of [`GetNetNotional`](https://github.com/dydxprotocol/v4-chain/blob/6d7544d23fd9fc20657089d70c616f8882f675ea/protocol/x/perpetuals/keeper/perpetual.go#L744) is signed, so it can't be cast to a `uint64` right?

Reply:
Will update to take the absolute value of notionalQuoteQuantums